src/app/shared/dockstore.model.ts

```diff
@@ -25,6 +25,9 @@ export class Dockstore {
   // Discourse URL MUST end with a slash (/)
   static readonly DISCOURSE_URL = 'http://localhost/';
 
+  // Google Shortener API key
```

Comment:
Maybe a link to docs for this, I think https://developers.google.com/url-shortener/v1/getting_started#APIKey